pulp_smash/tests/pulp3/file/api_v3/test_crd_publications.py

```diff
@@ -61,6 +65,7 @@ def test_01_create_publication(self):
             urljoin(self.publisher['_href'], 'publish/'),
             {'repository': self.repo['_href']}
         )
+        verify_number_tasks(len(call_report))
```

Comment:
If `len` must be used every time `verify_number_tasks` is called, then just call `len` inside `verify_number_tasks`. DRY.